app/models/user.rb

```diff
@@ -28,7 +28,7 @@ class User < ActiveRecord::Base
   CARDS_LIMIT_BEFORE_NOTIFY = 0
 
   def add_current_deck
-    decks.create(name: "Default deck")
+    decks.create(name: I18n.t('decks.default'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.